src/app/Table.elm

```diff
@@ -0,0 +1,29 @@
+module Table exposing (view, init, Msg)
```

Comment:
So, basically the nested architecture is just to do with state, isn't it?

I'm not nesting any state here.

I suppose it's also to do with messages, to a degree, too, if you want to keep them separated.